modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -1,270 +1,306 @@
 
 == API Access Requester
 
-The API access requester button component enables members to request access to an API for an application.
+The *API Access Requester* component enables community members to request access to an API for either a new or existing application.
+
+Community members can either request access for an existing client application and API instance, or request to create a new client application with access to the API by specifying the application name and description and choosing an API instance.
 
-Members can request access for an existing client application and API instance, or request to create a new client application by specifying the application name and description and choosing an API instance.
+You can specify values for the following parameters of the *API Access Requester*:
 
-API:: Choose the API.
-Button Border Radius:: Set the radius of the button border. A larger radius makes the button look more rounded.
-Button Align:: Set button alignment to left, right, or center.
-Show Modal Title Line:: Show the title line in the access request modal.
+API:: Specifies the API.
+Button Border Radius:: Selects the button border radius. A larger radius makes the button look more rounded.
+Button Align:: Specifies left, right, or center button alignment.
+Show Modal Title Line:: Shows the title line in the access request modal.
 
 // check description of Show Modal Title Line
 
 == API Card
 
-The API card component shows information about an API, including icon, name, and description.
+The *API Card* component shows information about an API, including icon, name, and description.
+
+You can specify values for the following parameters of the *API Card*:
 
-API:: Choose the API to show.
-Override URL:: When the user clicks the *API Details* button on the card, a URL opens. If this optional field is set, the URL in the field overrides the default URL. This could be used to direct users to a documentation or marketing page about the API, which would also have a link to the API's main page.
-Show Buttons:: Show the *API Details* button. If the button is not shown, users can click anywhere on the card to show the API details.
-Card Height:: Set the card height.
-Card Margin:: Set the amount of margin space around the card.
-Card Border:: Set the thickness of the card borders.
-Card Border Radius:: Set the radius of the card borders. A larger radius makes the cards look more rounded.
-Card Shadow:: Set the shadow beneath the card.
-API Details Button Label:: Set the label of the *API Details* button. Default is `API Details`.
-Learn More Button Label:: Set the label of the *Learn More* button. Default is `Learn More`.
+API:: Specifies the API shown.
+Override URL:: If set, overrides the default *API Details* button URL to the API main page in favor of linking to other information that you want the user to see first, such as documentation or marketing collateral.
```

Comment:
What is a button URL?

Reply:
When the user clicks the button they go to the button URL. Rewrote for clarity.